bindings/python/test/trajectory/model/test_tabulated.py

```
@@ -0,0 +1,332 @@
+    )
+    def test_calculate_state_at_failure(
+        self,
+        test_states: list[State],
+        interpolation_type: Interpolator.Type,
+    ):
+        tabulated: Tabulated = Tabulated(
+            states=test_states,
+            interpolation_type=interpolation_type,
+        )
+
+        with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)